_posts/2023-01-18-integrate-openfaas-with-aws-services.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Integrate OpenFaaS with AWS services"
+description: "Trigger OpenFaaS functions based upon many different types of events generated in AWS using an AWS SNS subscription."
+date: 2023-01-18
+image: /images/2022-trigger-functions-from-postgres/background.jpg
+categories:
+- aws
+- sns
+- functions
+- triggers
+author_staff_member: han
+---
+Trigger OpenFaaS functions based upon many different types of events generated in AWS using an AWS SNS subscription.
+
+## Introduction
+
```

Comment:
One other thing to mention is that SNS could be used to communicate to OpenFaaS Functions from existing microservices or systems on AWS, or even to get messages from partners where you give them access to a queue  with publish-only access.